eland/operations.py

```diff
@@ -472,13 +495,26 @@ def _unpack_metric_aggs(
 
                     # Pull multiple values from 'percentiles' result.
                     if es_agg[0] == "percentiles":
-                        agg_value = agg_value["values"]
-
-                    agg_value = agg_value[es_agg[1]]
-
+                        agg_value = agg_value["values"]  # Returns dictionary
```

Comment:
We're not using `es_agg[1]` any more? Is there a reason for that?

Reply:
For `median`/ `quantile` called in `df.agg[...]`
since es_agg would hold `('percentiles', [50.0])` 
So doing `agg_value = agg_value["50.0"]` would look more readable than 
`agg_value = agg_value[es_agg[1][0]]`

That `agg_value = agg_value[es_agg[1]]` is being used for other aggs below 😃

Reply:
For `median`/ `quantile` called in `df.agg[...]`
since es_agg would hold `('percentiles', [50.0])` 
So doing `agg_value = agg_value["50.0"]` would look more readable than 
`agg_value = agg_value[es_agg[1][0]]`

That `agg_value = agg_value[es_agg[1]]` is being used for other aggs below 😃